garage/contrib/ros/envs/sawyer/README.md

```diff
@@ -205,7 +209,15 @@ $ source activate garage
 **Make sure that $GARAGE is in your $PYTHONPATH** \
 Ex.
 
-    $ python $GARAGE/contrib/ros/envs/example_launchers/trpo_gazebo_sawyer_pnp.py
+    $ python $GARAGE/garage/contrib/ros/envs/example_launchers/trpo_gazebo_sawyer_pnp.py
 ## Troubleshooting
 ### ImportError: import cv2 when running training script
 Make sure your are using opencv-python installed in garage conda env.
+### Need to use boost_python-py35
+    - See https://github.com/osmcode/pyosmium/issues/52
+    - We need boost_python-py35
+    ```bash
+    # quick and dirty workaround
+    rm /usr/lib/x86_64-linux-gnu/libboost_python.so
+    ln -s /usr/lib/x86_64-linux-gnu/libboost_python-py35.so /usr/lib/x86_64-linux-gnu/libboost_python.so
```

Comment:
The latest conda environment is based on python36. Will this work?

Reply:
Yea, it works so far.